source/js/README.md

```diff
@@ -1,152 +1,592 @@
-# Astronomy Engine (JavaScript)
```

Comment:
This one got affected because I didn't find the documentation of `jsdoc2md templates` to include everything what is now outside the namespace.

Reply:
nvm, I figured it out searching again :D